diff --git a/src/app/api/orders/route.ts b/src/app/api/orders/route.ts
index 2737928..20b5e29 100755
--- a/src/app/api/orders/route.ts
+++ b/src/app/api/orders/route.ts
@@ -228,10 +228,13 @@ export async function POST(request: Request) {
       '',
       'Mohon infokan ongkos kirim Instan/Sameday beserta total transfer.',
       '',
-      '*Transfer ke:*',
+      '*Transfer ke salah satu rekening:*',
       'Bank Mandiri a.n. Sari Giyanti',
       'No. Rek: 1040004130840',
       '',
+      'Bank BNI a.n. Sari Giyanti',
+      'No. Rek: 1901349111',
+      '',
       'Terima kasih.',
     ].join('\n');
     const whatsappUrl = `https://wa.me/${adminPhone}?text=${encodeURIComponent(message)}`;
